Updating docker docs (#11291)
Automattic/jetpack: docker/README.md

* [Yarn](https://yarnpkg.com/) — please make sure your version is higher than v1.3: `yarn --version`
* Optionally [Ngrok](https://ngrok.com) client and account or some other service for creating a local HTTP tunnel. It’s fine to stay on the free pricing tier with Ngrok.

Install prerequisites; you will need to open up Docker to install its dependencies.

> Clone the repository:

Reviewer: ```suggestion

Start by cloning the Jetpack repository:
```

## Using Ngrok with Jetpack

To be able to connect Jetpack you will need a domain - you can use [Ngrok.com](https://ngrok.com/) to assign one.

If you use one-off domains, you'll have to re-install WordPress and re-connect Jetpack each time you close Ngrok (thus losing your randomly assigned domain). That's perfectly fine for quick testing or lightweight development. You can use [other similar services](https://alternativeto.net/software/ngrok/) as well.

> If you're developing Jetpack often you'll want to reserve a domain you can keep using. If you create an account by signing up via Google and using your a8c Google account you'll be added to our business tier plan which will enable you to re-use domains, reserve your custom domains and reserve TCP ports.

Reviewer: ```suggestion
If you're developing Jetpack often you'll want to reserve a domain you can keep using.

If you are an Automattician, sign up on Ngrok.com using your a8c Google account; you'll be automattically added to the Automattic team. That will enable you to re-use domains, reserve your custom domains and reserve TCP ports.
```

> Once you’ve joined the team, [reserve yourself a permanent domain here](https://dashboard.ngrok.com/reserved).

Reviewer: ```suggestion
[Go to this page to reserve a permanent domain](https://dashboard.ngrok.com/reserved).
```

> Once you’ve done that, follow [these steps](https://ngrok.com/download) to download and set up ngrok. However, instead of step four, edit your [config file](https://ngrok.com/docs#default-config-location) or use `-config=your_config_file.yml` argument with `ngrok` to use your configuration file.

Reviewer: I would remove `or use `-config=your_config_file.yml` argument with `ngrok` to use your configuration file.` to keep things simple.

```suggestion
Once you’ve done that, follow [these steps](https://ngrok.com/download) to download and set up ngrok. However, instead of step four, edit your [config file](https://ngrok.com/docs#default-config-location) as explained below:
```

> Edit your config file so it looks like this:

Reviewer: ```suggestion

```

```
authtoken: YOUR_AUTH_TOKEN # This should already be here
region: eu # only needed for subdomains in Europe
tunnels:
  jetpack:
    subdomain: YOUR_RESERVED_SUBDOMAIN # without the .ngrok.io
    addr: 80

>     proto: http```

Reviewer: ```suggestion
    proto: http
```
